src/common/components/AggregationFilter.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Immutable from 'immutable';
+
+import CheckboxAggregation from './CheckboxAggregation';
+import RangeAggregation from './RangeAggregation';
+
+class AggregationFilter extends Component {
```

Comment:
in general is more practical to use something like:

```
_getRangeAggregation() {
    return (
    );
}

_getCheckboxAggregation() {
    return (
    );
}

render() {
   let content = range ? this._getRangeAggregation() : this._getCheckboxAggregation()

}
```

Reply:
Do we need to have it as a component class? it doesn't have any state

Reply:
well that's what I thought too but at some point I think we said  "let's have everything as class to be consistent".